src/store-service.js

```diff
@@ -104,6 +107,16 @@
                 .map((store) => store.id);
         }
 
+        getPreviousClosedWindowNames() {
+            return (this.storage || [])
+                .filter((store) => store.closed > 0)
+                .map((store) => store.id);
+        }
+
+        getPreviousClosedWindows() {
+            return this.getPreviousClosedWindowNames().map((name) => this.open(name).store);
```

Comment:
This feels like a roundabout way to get the object you actually want. We know this object exists in the store ready, so I don't think we need to go to the trouble of creating a `StoreWrapper` by injecting the store object, just to fetch the object back off of it.